apps/crn-server/test/data-providers/research-outputs.data-provider.test.ts

```diff
@@ -83,7 +83,7 @@ describe('ResearchOutputs data provider', () => {
       );
       const expectedResult = getResearchOutputDataObject();
 
-      expect(result).toEqual(expectedResult);
+      expect(result).toEqual({ ...expectedResult, published: true });
```

Comment:
why is it not part of the fixture? You've added it to the DataObject so it's best to have it within the fixture? I guess it's because it was marked as undefined - well in my opinion it should not be undefined

Reply:
fixed here [review: make published mandatory](https://github.com/yldio/asap-hub/pull/2667/commits/7ab15402dfc9a32c8bb9335ea9e6e2bf4110fd82)